www/js/share/components/accounts/accounts.template.html

```diff
@@ -4,7 +4,7 @@
     </div>
     <select class="account-select" ng-change="vm.updateAccount(vm.selectedAccount)" ng-model="vm.selectedAccount">
             <option ng-repeat="a in vm.accounts" value="{{a.token}}" ng-selected="{{a.token == vm.selectedAccount}}">
-            {{a.id}} {{a.currency == null ? "" : "(" + a.currency + ")"}}
+            {{a.id}} {{!a.currency ? "" : "(" + a.currency + ")"}}
```

Comment:
@morteza-binary when the currency is not set it's equal to "" so the second condition is true and the first one is false so it adds the parentheses to account name.